Makefile_gnu

```diff
@@ -1,9 +1,11 @@
-ifdef MPI
-FC = mpif90
-CC = mpicc
-CXX = mpicxx
+ifeq ($(MPI),1)
+	FC := $(if $(FC),$(FC),mpifort)
```

Comment:
What is the error you're getting? I just tested on my linux machine, both with and without MPI, and using python setup.py install, as well as the the old make, and don't get any errors.

Reply:
It was something that occurred in the [travis build](https://travis-ci.org/PolyChord/PolyChordLite/jobs/608932210), although it was the `make MPI=1 gaussian` that was failing, rather than the setup.py

Reply:
Ah, it seems on linux $(CXX) defaults to g++. I think I fixed it.